maestrowf/maestro.py

```diff
@@ -136,7 +136,7 @@ def run_study(args):
     environment.add(Variable("OUTPUT_PATH", output_path))
 
     # Now that we know outpath, set up logging.
-    setup_logging(args, output_path, spec.name)
+    setup_logging(args, output_path, spec.name.replace(" ", "_"))
```

Comment:
Just a thought that came to mind -- `.lower()` on the name might also be useful.

Reply:
Sure, I can add that.